flags.go

```diff
@@ -178,6 +231,28 @@ func BackendFromJSONFile(filename string) Backend {
 // Consul key/value storage.
 func RefreshFlags(backend Backend) error {
 
+	updateableB, ok := backend.(UpdatableBackend)
+	if ok {
+		t, err := updateableB.UpdateTime()
+		if err == nil {
+			staleness := time.Since(t)
+			stale := staleness > STALENESS_THRESHOLD
+			//histogram of staleness
+			stats.Histogram("goforit.flags.cache_file_age_s", staleness.Seconds(), nil, .1)
+			//gauge
+			var gauge float64
+			if stale {
+				gauge = 1
+			}
+			stats.Gauge("goforit.flags.cache_file_stale", gauge, nil, 1)
```

Comment:
Let's just log a message for staleness here, same as above.

Reply:
Have added logging